.changeset/dirty-singers-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/mdx': patch
```

Comment:
shouldn't this be a minor?

Reply:
Breaking change -> major

I also think this should be a **minor**.

Reply:
I'm confused. There's no breaking changes, and `@astrojs/mdx` is v0. So breaking changes -> minor, features/fixes -> patch?